code/modules/mob/living/carbon/human/species_types/ethereal.dm

```diff
@@ -37,6 +37,7 @@
 	hair_color = "fixedmutcolor"
 	hair_alpha = 140
 	swimming_component = /datum/component/swimming/ethereal
+	possable_faculties = list(PSI_ENERGISTICS)
```

Comment:
ethereals should probably have a higher latency too

Reply:
adjusted base to 15 for now, Ethereals are at 25